### PR TITLE
Grammatical and typo corrections

### PR DESCRIPTION
**Pull request by famouswizard**
Correct grammatical errors and improve readability of the documentation.

### CODE REVIEW

File: src/developers/advanced_topics/persistent_storage.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ added `linera-db` for that purpose.
 
 ## Available persistent storage
 
-The persistent storage that are available right now are `RocksDB`, `DynamoDB`
+The persistent storage that is available right now includes are `RocksDB`, `DynamoDB`,
```
Comment (ma2bd) [inline src/developers/advanced_topics/persistent_storage.md:9]: remove the second "are" ?

Comment (famouswizard) [reply src/developers/advanced_topics/persistent_storage.md:9]: > remove the second "are" ?

Sure.

Comment (famouswizard) [reply src/developers/advanced_topics/persistent_storage.md]: Updated.

**Review by ma2bd — APPROVED** [review]

**Comment by famouswizard** [comment]
Please merge it.

**Comment by famouswizard** [comment]
[ma2bd](https://github.com/ma2bd) please merge it!

**Pull request merged**